perllib/FixMyStreet/Cobrand/Default.pm

```diff
@@ -6,8 +6,6 @@ use warnings;
 use FixMyStreet;
 use FixMyStreet::DB;
 use FixMyStreet::Geocode::Address;
-use FixMyStreet::Geocode::Address::OSM;
-use FixMyStreet::Geocode::Bing;
 use FixMyStreet::Geocode::OSM;
```

Comment:
Need to `use FixMyStreet::Geocode::OSPlaces;` (and probably `Bing` too, in case cobrands want to use that) here.

Reply:
Hmm, I should have this in `FixMyStreet::Geocode` itself, shouldn't I, so it can use the plugged in geocoders, like the `string` lookup